### DISCUSSION TITLE
How to show alert/toast notification

### DISCUSSION THREAD

**Discussion by ryzam** [Budibase/budibase#3396] [Help] [answered]
Hi guys, sorry if this topic has been discussed but I unable to find  how to show custom message as alert/toast notification.

**Comment by mjashanks** [comment, marked as answer]
We do not have this feature. Depending on what you are trying to do, you could conditionally show & hide a container.

**Reply by mroziken** [reply]
@mjashanks 

I would like to do the following:
1. Setup webhook trigger to insert records into notification table - This works for me already
2. Once new record is added to notification table - set a app state newAlert = True - This does not work for me
3. Based on the app state newAlert hide or show notification icon - This works for me.

My questions are:
a) Is it possible to set app state from automation action ? How?
b) If the answer for the above is: No .. does anybody know how I can control display of an icon based on some trigger?
